### ISSUE TITLE
Global Access to Research Software

### ISSUE THREAD

**Issue by KoenVermeir** [sparcopen/doathon#34]
_Confused? New to Github? Visit the [GitHub help page on our site for more information](http://doathon.opencon2017.org/)!_

[//]: # "=======================
Even if you know Github well, we suggest you read this. Anything between these lines you can leave or delete, as they won't display anyway when you post (you can check this via Preview changes). They're here to help you complete issues quickly and in a way that will help other participants. If you're posting a new project, or challenge. We suggest you fill out the Google Forms first.
============================"

## At a glance

[//]: # "=======================
Please paste the metadata you received after submitting your project or challenge in your Google Form exactly as we sent it to you. You can delete what's there now, it's just there
============================"

* **Submission Name:** _Global Access to Research Software_
* **Contact Lead:** _koen.vermeir.gya@gmail.com_
* **External Site Link (optional - primarily for projects):** _https://globalyoungacademy.net/activities/global-access-to-research-software/_
* **Region**: _#Global_
* **Issue Area:** _#OpenAccess, #OpenResearch_
* **Issue Type:** _#Project_
* **Types of Support Needed:** _#Advocacy_and_Policy, #Communications, #Community_GrassrootsOrganizing_ 

## Description
[//]: # "=======================
Insert a paragraph providing more context for your project or challenge focuses on. For project leads, this is a good place to give some broader context about your project—beyond the scope of the do-a-thon. If you're posting a challenge, this is a good chance to say how the problem arise or why it feels relevant to you.
============================"
Recent research has shown that researchers in low- and middle income countries overwhelmingly rely on proprietary software, but they often do not have the means to acquire this software. Furthermore, the desired Research Software is highly field-specific and diverse.

On the other hand, they are hardly aware of the existence and possibilities of Free and Open Source Software (FOSS) alternatives to proprietary Research Software. 
 
As a result, a survey has shown that the lack of access to software significantly affects research content and networks. The lack of access to Research Software thus obstructs equal research opportunities and the reproducibility of research results.

How do we raise awareness of field-specific Free and Open Source Software (FOSS)  in low- and middle income countries? What are the right networks and strategies for communication and dissemination?

## What are we working on during the do-a-thon? What kinds of support do we need?
[//]: # "=======================
For those leading projects, please give some more information about what type of support you are specifically looking to get done during the do-a-thon day.
Note: Challenge leads will not need to fill out this section and can remove it.
============================"
#Advocacy_and_Policy, 
#Communications, 
#Community_GrassrootsOrganizing_
- Brainstorming on how do we raise awareness of field-specific Free and Open Source Software (FOSS)  in low- and middle income countries? 
- Finding the right networks and strategies for communication and dissemination of awareness and knowledge of FOSS.
 

##  How can others contribute?
[//]: # "======================= 
Please say what the best way to contribute to the project or challenge is, sometimes that will just be 'lets discuss here' or 'Ive started a Google doc'. 
If you are a challenge lead, give some context on what design thinking tools you will be using, and how other folks can update their ideas onto the thread. 
If you are a project lead, and you already have clear ways people can contribute it might be worth linking to them here.
Language: If your project is regionally based in a non-English-speaking region, clarify here what language you and contributors will primarily be communicating in. 
If you're leading a project or challenge participating remotely: Use this space to let people know that this is a remote project and that you are not 'in the room' in Berlin. Let other participants know what the best way to get in touch with you, where the work will happen, and where any updates or outputs will go. 
If you are at the in-person meeting in Berlin: Be as inclusive as possible to those outside the room.  Use this space to give clear instructions to those participating in the do-a-thon remotely on how they can keep up to date and contribute. ============================"

- Brainstorming on how do we raise awareness of field-specific Free and Open Source Software (FOSS)  in low- and middle income countries? 
- Sharing the right networks and strategies for communication and dissemination of awareness and knowledge of FOSS.
- Advocating the uptake of FOSS by researchers in richer countries, as this may increase visibility elsewhere too and it will downplay compatibility concerns.
- Grassroots global advocacy of FOSS.

[//]: # "=======================
You're ready to post!!! After posting your issue, the real work begins. Next, you might want to:
   Tweet a link to this issue with #opencon so others can join in
   Make another issue to involve people in your work - remember to use your metadata
   Come back from time to time and update the community on your project.
You'll get an email update whenever someone interacts with your issue.
============================"

_This post is part of the OpenCon 2017 Do-A-Thon. Not sure what's going on? [Head here](http://doathon.opencon2017.org/)._

**Comment by npscience** [comment]
Some notes:

* This doesn't appear to only be a LMIC problem: a researcher in US found students there only use the tools provided by the institution too, and aren't aware of others. [More info on this and an idea](https://github.com/npscience/eLife-at-MozFest2017/issues/12).
* David Perez is trying to gather a list of good tools for collaborative research online and with an annual conference (maybe not an option for all). You can use and contribute to his resources on GitHub - [more info](info at https://github.com/npscience/eLife-at-MozFest2017/issues/17).
* The FOSS community seemingly needs support everywhere, I'd love to hear about it from your context too.
* NumFocus could be a good resource?

**Comment by KoenVermeir** [comment]
thanks for your comment! I'll look into yhe tools you mention. The LMIC problem is that the don't have the means to use traditional software, but just aren't aware about FOSS, which is quite dramatic for their research.
I think bottom up activism will be key. There should be FOSS ambassadors at different universities, or somethong like it

**Comment by asbtariq** [comment]
Thanks Koen @KoenVermeir for opening this here and thanks Naomi @npscience for your feedback.
You are quite correct.  In fact when the Working Group was formed at the Global Young Academy, it was named Research Software for Developing Countries. It was changed to Global Access to Research Software when several members from developed countries felt they too face barriers in access.
However the survey Koen mentioned focused on LMICs or developing countries and it documented not only the low take-up, which is a global issue, but the low awareness itself of FOSS.

**Comment by leenyahodza** [comment]
I think this could work mostly for early career researcher too. FOSS are useful in advancing research. My question is, do we need to create a list first before we look at advocacy? I'm really interested to learn from this group.

**Comment by livmar2000** [comment]
hey, are you seated somewhere?

**Comment by israb12** [comment]
@KoenVermeir, Talking of networks, in Africa you might be interested to check out the Next Einstein Forum on http://nef.org/

**Comment by solomon3m** [comment]
It is really very timely  and relevant project.  I would like to be parts of the project and support on spreading the available open source research software to my African colleagues.

**Comment by asbtariq** [comment]
Good to see continued / renewed interested in GARS.  This is very rewarding!

Just an anecdotal point - GARS (originally as RSDC) was proposed in Berlin in 2011 at the first AGM or GA of the GYA.

**Comment by KoenVermeir** [comment]
Hi all,

we're physically present in the Goethe room (the main conference room); We'll continue after lunch and the lightning sessions.

Thanks for joining in!
--Koen

**Comment by KoenVermeir** [comment]
Hi all: to provide some more background (it was in the original OpenCon application, but doesn't appear here on Github):

this project started with a GYA survey and research report on the uptake of FOSS:  https://globalyoungacademy.net/activities/global-access-to-research-software/
 
now we want to work with the results and find ways to raise awareness of FOSS, especially in LMIC countries

**Comment by KoenVermeir** [comment]
Hi all,
we're starting again at 4 PM in the Goethe room.
looking forward to seeing you!

**Comment by KoenVermeir** [comment]
Quiet for a moment. Focusing on Open Conferences

**Comment by leenyahodza** [comment]
Hi
I could not locate you so I joined the#60 for librarians. I was interested in the FOSS but I struggled to find the group.

Kind Regards,

Lena Nyahodza

[http://www.uct.ac.za/images/uct.ac.za/about/logo/logocircless.gif]

Lena Nyahodza
Scholarly Communications Librarian
University of Cape Town Libraries
Email: lena.nyahodza@uct.ac.za<mailto:lena.nyahodza@uct.ac.za>
Tel: +27 (0)21 650 5469
Research Wing: Level 5, Office 7
Chancellor Oppenheimer Library

**Comment by KoenVermeir** [comment]
yes, still the same place: Goethe room, table at the back, far left
